### PR TITLE
One by AOL - Add manual impression tracking

### PR DESCRIPTION
**Pull request by chauduyphanvu1**

### CODE REVIEW

File: OnebyAOL/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Changelog
+  * 6.8.1.4
```
Comment (rsatrasala) [inline OnebyAOL/CHANGELOG.md]: Changelog updates same as Facebook

Comment (rsatrasala) [inline OnebyAOL/CHANGELOG.md]: Why are other format changes not included?

Comment (chauduyphanvu1) [reply OnebyAOL/CHANGELOG.md]: Reason was given on `EPSO-13969` but in short, the AOL SDK does not have a corresponding banner callback for an "ad show" event to map ours to (https://github.com/mopub/mopub-android-mediation/blob/master/OnebyAOL/com/mopub/mobileads/MillennialBanner.java#L141-L206).

Comment (rsatrasala) [reply OnebyAOL/CHANGELOG.md]: ah got it.

**Pull request merged**